Taskfile.yml

```diff
@@ -158,35 +160,62 @@ tasks:
     env:
       DOCKER_COMPOSE_FILES: "{{ .DOCKER_COMPOSE_FILES_CI }}"
 
-  ci:behat:
-    desc: Run Behat functional tests
+  ci:cypress:
+    desc: Run Cypress functional tests
     cmds:
-      - task dev:cli -- vendor/bin/behat
+      - docker compose {{ .DOCKER_COMPOSE_FILES_CI }} run node-chrome yarn install --frozen-lockfile
+      - task dev:cli -- drush user:password $CYPRESS_DRUPAL_USERNAME $CYPRESS_DRUPAL_PASSWORD
+      - docker compose {{ .DOCKER_COMPOSE_FILES_CI }} run -e CYPRESS_DRUPAL_USERNAME=$CYPRESS_DRUPAL_USERNAME -e CYPRESS_DRUPAL_PASSWORD=$CYPRESS_DRUPAL_PASSWORD cypress
     env:
-      DOCKER_COMPOSE_FILES: "{{ .DOCKER_COMPOSE_FILES_CI }}"
+      CYPRESS_DRUPAL_USERNAME: admin
+      CYPRESS_DRUPAL_PASSWORD: admin
+
+  ci:cypress:local:
+    desc: Run Cypress functional tests locally
+    cmds:
+      - yarn install --frozen-lockfile
+      - task dev:cli -- drush user:password $CYPRESS_DRUPAL_USERNAME $CYPRESS_DRUPAL_PASSWORD
+      - http_proxy=http://localhost:{{ .WIREMOCK_HTTP_PORT }} https_proxy=http://localhost:{{ .WIREMOCK_HTTPS_PORT }} npx cypress open
+    env:
+      CYPRESS_BASE_URL: http://localhost:{{ .CYPRESS_BASE_PORT }}
```

Comment:
Gessuzz! This has been tricky to solve :)

Reply:
Yes 😅